lms/djangoapps/instructor/views/legacy.py

```diff
@@ -631,17 +631,21 @@ def __init__(self):
         self.grades = {}
         self._current_row = {}
 
-    def _add_grade_to_row(self, component, score):
+    def _add_grade_to_row(self, component, score, possible=None):
         """Creates component if needed, and assigns score
 
         Args:
             component (str): Course component being graded
             score (float): Score of student on component
+            possible (float): Max possible score for the component
```

Comment:
The max possible score is always an integer, since it is the number of problems.  Do we gain anything by asserting that here?